src/node_perf.cc

```diff
@@ -525,25 +557,76 @@ bool ELDHistogram::RecordDelta() {
     }
   }
   prev_ = time;
-  return ret;
+}
+
+void ELDHistogram::RecordPreciseDelayStart() {
+  prev_ = uv_hrtime();
+}
+
+void ELDHistogram::RecordPreciseDelayStop() {
+  if (prev_ == 0) {
+    return;
+  }
```

Comment:
Super minor nit... as of late we've been moving away from `{` `}`'s for single line ifs.
```suggestion
  if (prev_ == 0)
    return;
```

Reply:
Is it part of official code style for Node.js? `make lint` didn't catch this.

If there any way to persuade, I'd insist that we keep parens even though (as many of you certainly know and remember) I was a proponent of omitting them. That apple kernel bug that was fixed some time ago has convinced me that omitting parens is error prone. I don't have a link to the kernel fix anymore, but it looked like this:

```C
if (something)
if (something_else)
  do_something();
```

Forgetting to put an `if` body would result it in compiler interpreting the code in likely undesirable way.

That being said, I don't insist on changing existing code. Still it'd be great to write new code differently!